### PR TITLE
Added strict-mode for CI servers

### LINKED ISSUE #55: Strict-Mode for Ci-Server
**Is your feature request related to a problem? Please describe.**
Currently when the snapshot is not existent snapshooter creates a new one without the test failing. This behavior is nice if you are developing on your local machine and speeds op development. However, if the test is run on a CI-Server, the test should fail if the snapshot does not exist.

**Describe the solution you'd like**
It should be possible to have an environment variable like `SNAPSHOOTER_STRICT_MODE` set which tells snapshooter to fail if the snapshot does not exist.

### PR DESCRIPTION
**Pull request by michaelstaib**
Fixes #55

### CODE REVIEW

File: src/Snapshooter.Xunit.Tests/SnapshotTests.cs [hunk 1]
```diff
@@ -24,7 +24,23 @@ public void Match_FactMatchSingleSnapshot_GoodCase()
             // act & assert
             Snapshot.Match<TestPerson>(testPerson);
         }
-        
+
+        [InlineData("on")]
+        [InlineData("true")]
+        [Theory]
+        public void Strict_Mode_No_Snapshot_Found(string value)
```
Comment (RohrerF) [inline src/Snapshooter.Xunit.Tests/SnapshotTests.cs]: Does not match out naming pattern

File: src/Snapshooter.Xunit.Tests/SnapshotTests.cs [hunk 2]
```diff
@@ -24,7 +24,23 @@ public void Match_FactMatchSingleSnapshot_GoodCase()
             // act & assert
             Snapshot.Match<TestPerson>(testPerson);
         }
-        
+
```
Comment (RohrerF) [inline src/Snapshooter.Xunit.Tests/SnapshotTests.cs]: test if not thrown when set to false or off

File: src/Snapshooter/Exceptions/NoSnapshotFoundException.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Snapshooter.Exceptions
+{
+    /// <summary>
+    /// Exception thrown if something went wrong during snapshot assertion.
+    /// </summary>
+    public class NoSnapshotFoundException : SnapshotTestException
```
Comment (RohrerF) [inline src/Snapshooter/Exceptions/NoSnapshotFoundException.cs]: How about SnapshotNotFoundException instead of NoSnapshotFoundexception

**Comment by RohrerF** [comment]
What about the documentation?

File: src/Snapshooter/Exceptions/NoSnapshotFoundException.cs (hunk 3 above)
Comment (michaelstaib) [reply src/Snapshooter/Exceptions/NoSnapshotFoundException.cs]: I agree, that name is better!

**Pull request merged**